ql/src/java/org/apache/hadoop/hive/ql/exec/FetchTask.java

```diff
@@ -76,7 +76,7 @@ public void initialize(QueryState queryState, QueryPlan queryPlan, DriverContext
         TableScanOperator ts = (TableScanOperator) source;
         // push down projections
         ColumnProjectionUtils.appendReadColumns(
-            job, ts.getNeededColumnIDs(), ts.getNeededColumns());
+            job, ts.getNeededColumnIDs(), ts.getNeededColumns(), ts.getNeededGroupPaths());
```

Comment:
"Group" is a uniquely Parquet terminology. Would you mind replacing it with the Hive lingo?